src/lib/error/bad-data-error.ts

```diff
@@ -16,7 +16,7 @@ class BadDataError extends UnleashError {
         errors?: [ValidationErrorDescription, ...ValidationErrorDescription[]],
     ) {
         const topLevelMessage =
-            'Request validation failed: your request body contains invalid data' +
+            'Request validation failed: your request body or params contain invalid data' +
```

Comment:
we throw this error when query or path params are incorrect so the error message should be more generic